### ISSUE TITLE
Does HD1k really help?

### ISSUE THREAD

**Issue by askerlee** [zacjiang/GMA#18]
I was using HD1k when doing Sintel finetuning, just as GMA does. I'm surprised that it only consists of grayscale images; that means there's a big domain gap between HD1k and other training sets. Wonder if I remove HD1k, would the model be trained better? My training without HD1k is ongoing, and seems the loss on the training data is much smaller, and accuracy is higher. Would update when it finishes.

**Comment by askerlee** [comment]
Seems eventually the performance is very similar. Anyway HD1k is just a small part of Sintel fine-tuning data.
